docs/Users/Install.rst

```diff
@@ -153,6 +153,11 @@ If you want to install coala to an alternate location you can e.g. call
 options are documented on
 https://docs.python.org/3.3/install/#alternate-installation.
 
+..note::
+
+    Make sure you do not use any proxy at installation
```

Comment:
because pip can't download stuff. Proxies *always* are a probklem.

Sincerely,

Lasse Schuirmann

contact@viperdev.io
http://viperdev.io/

2016-02-26 19:15 GMT+01:00 Sudheesh Singanamalla <notifications@github.com>:

> In docs/Users/Install.rst
> <https://github.com/coala-analyzer/coala/pull/1719#discussion_r54279730>:
>
> > @@ -153,6 +153,11 @@ If you want to install coala to an alternate location you can e.g. call
> >  options are documented on
> >  https://docs.python.org/3.3/install/#alternate-installation.
> >
> > +..note::
> > +
> > +    Make sure you do not use any proxy at installation
>
> Why exactly is a proxy a problem ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1719/files#r54279730>.
>